server/pbenchinacan/etc/pbench-server/pbench-server.cfg

```diff
@@ -15,16 +15,16 @@ roles = pbench-results
 
 [Indexing]
 index_prefix = container-pbench
-uri = http://localhost:9200
+uri = https://localhost:9200
 
 [database]
-uri = postgresql://pbenchcontainer:pbench@localhost:5432/pbenchcontainer
+uri = postgresql://pbenchcontainer:pbench@localhost:5432/pbenchcontainer?sslmode=require&sslcert=/srv/pbench/postgres.pem
```

Comment:
Why provide `sslcert`?  Why not just put the `postgres.pem` file where the PostgreSQL client expects to find it?